File: es/buscas/java/draft.java
import java.util.*;
import java.util.stream.Collectors;
import java.util.stream.IntStream;

public class draft {
    static Scanner scan =  new Scanner(System.in);
    static ArrayList<String> consul = new ArrayList<>();
    static ArrayList<String> input = new ArrayList<>();

    public static int countCountain(String str){
        int cont = 0;
        for(int i=0; i<input.size(); i++){
            if(str.equals(input.get(i)))
                cont++;
        }

        return cont;
    }
    
    public static void main(String args[]) {
        int qtdInput = scan.nextInt();
        scan.nextLine();
        input = IntStream.range(0, qtdInput).mapToObj(i -> scan.next()).collect(Collectors.toCollection(ArrayList::new));
        int qtdConsul = scan.nextInt();
        scan.nextLine();
        consul = IntStream.range(0, qtdConsul).mapToObj(i -> scan.next()).collect(Collectors.toCollection(ArrayList::new));
        String str = "";
        for(String s : consul){
            str += countCountain(s) + " ";
        }
        str = str.trim();

        System.out.println(str);
    

        

    }
}

File: es/diagonal/src/go/draft.go
package main

import "fmt"

func print(cont int, caracter []rune) {
	if cont == len(caracter) {
		return
	}

	str := ""
	for range cont {
		str += " "
	}

	str += string(caracter[cont])
	fmt.Println(str)
	cont++
	print(cont, caracter)
}

func main() {
	var line string
	fmt.Scan(&line)
	caracter := []rune(line)
	print(0, caracter)
}
